src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -1,21 +1,30 @@
 package org.broadinstitute.dsde.workbench.avram.api
 
-import java.util.logging.Logger
 
 import com.google.api.server.spi.config.{Api, ApiMethod}
+import java.sql.Timestamp
+import java.util.logging.Logger
 import javax.servlet.http.HttpServletRequest
-import org.broadinstitute.dsde.workbench.avram.Avram
 import org.broadinstitute.dsde.workbench.avram.dataaccess.SamUserInfoResponse
 import org.broadinstitute.dsde.workbench.avram.util.ErrorResponse
-import slick.jdbc.PostgresProfile.api._
-
 import scala.beans.BeanProperty
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.Duration
-import scala.concurrent.{Await, Future}
+
 
 case class Pong()
-case class Now(@BeanProperty message: String)
+case class Collection(@BeanProperty name: String,
```

Comment:
I am wondering if these case classes should be moved out of the routes and into a `model` package.

Reply:
I wonder if they can be merged with the `*Record` case classes. I have mixed feelings about that because they are technically different things, but they are _so_ similar that the strict separation might be less valuable than the reduction of code.

Reply:
I would keep the Record classes separate for now - record classes have `id`s, which response classes don't. And in the future we may want to keep track of other fields in our record classes and not our response classes.